servers/rendering/renderer_scene_cull.cpp

```diff
@@ -924,8 +1003,58 @@ void RendererSceneCull::instance_set_transform(RID p_instance, const Transform3D
 	Instance *instance = instance_owner.get_or_null(p_instance);
 	ERR_FAIL_NULL(instance);
 
-	if (instance->transform == p_transform) {
-		return; //must be checked to avoid worst evil
+	if (!(_interpolation_data.interpolation_enabled && instance->interpolated) || !instance->scenario) {
```

Comment:
```suggestion
	if (!_interpolation_data.interpolation_enabled || !instance->interpolated || !instance->scenario) {
```
Equivalent but more readable